literature/ocko_1997_halide_55/ocko_1997_halide_55_p6_6a.yaml

```diff
@@ -0,0 +1,102 @@
+# This file describes a figure from a publication containing electrochemical data.
+# For the `echemdb/website` project, the source, curator and electrochemical system is mandatory.
+# Figure description is optional. It is created automatically by the cv module of svgdigitizer
+
+source:
+    version: 1
+    doi: 10.1557/PROC-451-55 #only the number without preceeding doix or https
+    bib: Ocko_1997_Halide_55 # AuthorName_YYYY_FirstWordTitle_pageNr. No prepositions. Fingers crossed that there is no duplicate.
+    figure: 6a # depending on the subfigure labels use 1, 1b, or 1_3
+    curve: 0.05 M NaBr # use a unique short description to identify the curve in the figure, i.e.. label, color, etc
+    
+curator:
+    version: 1
+    name: Markus Schilling
+    orcid: https://orcid.org/0000-0003-2684-3711
+    digitized: 2021-10-18 # YYYY-MM-DD
+    
+figure description: #This section will be updated by module `CV`
+    version: 1
+    linked measurements: Synchrotron X-ray scattering # FTIR, SXRD, Ring Disc - indicate if any other operando methods were shown in the same plot, which is linked to the data

```

Comment:
SXRD ?

Reply:
For me, SXRD usually refers to single crystal XRD, but can be also Surface XRD. Synchrotron XRD as SXRD would be new for me. But I am fine with all options.

Reply:
SXRD refers to "surface  X-ray diffraction" which is often conducted on single crystal electrodes. So the "S" does not stand for "synchrotron".

Reply:
Changed to Synchrotron XRD